pkg/auth/auth_handler.go

```diff
@@ -77,6 +77,13 @@ func (a *AuthHandler) getValidationToken(token *jwt.Token) (interface{}, error)
 	return key, nil
 }
 
+func getError(err error) error {
```

Comment:
Should probably be in https://github.com/openshift/assisted-service/blob/master/internal/common/error_utils.go
Something like https://github.com/openshift/assisted-service/blob/master/internal/common/error_utils.go#L109
but it's ok now.

Reply:
Moved to error_utils